### DISCUSSION TITLE
Skinning - Combo Break Sound Changes

### DISCUSSION THREAD

**Discussion by Snowleopard-osu** [ppy/osu#13041] [Ideas]
**This is a proposal:**

Allow users to set what combo is the minimum required for the `combobreak.wav` to play.  I believe it is currently set to 20 by default.  It's possible this would be better as a game setting than part of `skin.ini`

**Alternative idea:**

Allow multiple combo break sounds using the standard `combobreak-{n}.wav` format other skinning objects follow.  The game would then either rotate through combo sounds or pick one randomly every time one should play.

**Why I want this:**

I got a new combo break sound that I really like, but if it gets a bit annoying if played back to back to back in a short time period.  This makes attempting hard songs where I'm going to miss frequently annoying.  This issue exists with more subtle sounds as well, it just became even more apparent with the sound I'm currently using.

While this is probably a textbook example of a "1st world problem", I think this would be a cool feature to support (maybe a Supporter only perk?)

**Comment by kyogoi** [comment]
> I think this would be a cool feature to support (maybe a Supporter only perk?)  


Should features that directly affect gameplay be supporter-only perks? Currently the precedent is that it's purely cosmetic, but I'm not sure if the standard has changed.

**Comment by peppy** [comment]
This is "purely cosmetic" but please don't try and decide what should be as supporter feature. In most cases I'm not interested in locking functionality that doesn't require extra *server-side* facilities behind supporter.

**Comment by Snowleopard-osu** [comment]
Definitely ok for this to be a feature anyone can use, and I would rather see it be available to all. I only added the supporter suggestion since some skinning features are already used as a supporter perk.

![EF154903-3DF5-404C-A45F-65C0BB13D3EB](https://user-images.githubusercontent.com/49334335/70680443-f8cde200-1c5d-11ea-8a27-5629f0477534.jpeg)

**Comment by Snowleopard-osu** [comment]
Saw that this issue was closed a few hours ago but with no comments or commits. Is that a “no” decision for implementing this feature?

**Reply by peppy** [reply]
We recently migrated all proposals to github discussions. The issue is closed but the discussion lives on for further discussion and upvoting.

**Reply by LittleEndu** [reply]
All issues that had the "proposal" label were closed and move to discussions system.

**Reply by Snowleopard-osu** [reply]
Ah ok that makes sense, thanks!

**Comment by IIIBlueberry** [comment]
I really the idea allow you to have multiple combo break sounds at different minimum combos.

something like combobreak-{n}-{n}.wav and combobreak: 20,75,300 on skin.ini
